### DISCUSSION TITLE
Several issues I need help with<title>

### DISCUSSION THREAD

**Discussion by davidgar026** [community/community#52532] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Initially, I wanted to get out of the folder that I was in and go back to the main folder which is "David Garcia". I can't run git branch -r or git branch -b which is to show me what all local and remote branch names. I'm doing this all in the terminal of VSCODE and this is what I  got:

David Garcia@SurfaceBook MINGW64 ~/vscode/proj1 (main)
$ git c:\"David Garcia"
> 
> 
> 
> 
> 
> 
> 
> 
> 
> reset
> quit
> git 
> reset
> 
> 
> 
> 
> 

Now I have another issue trying to get out of >. 

So I have several issues.

1) When I run git branch -b it gives me "error: unknown switch `b'" but when I run git branch -r it runs but nothing shows which I assume there is no branches created.

2) I cant get out of >

I was trying to change directories to see if the command git branch -b would work.
![11 04 2023_10 22 41_REC](https://user-images.githubusercontent.com/130368930/231196712-dad4064b-6dbc-4115-b012-88fda246d2bb.png)

**Comment by airtower-luna** [comment]
It looks like you're using Git Bash, and you opened a string, by typing first an escaped quote (`\"`) and then an unescaped one (`"`). So you're basically still in the command you started with `git c:\"David Garcia"`, with a few newlines since. You could close the string with another quote, but it's probably best to press Ctrl+C to cancel the whole thing. :sweat_smile: 

Also you can't change the directory with `git`, you need to use `cd` for that.

That leaves the question of how to write the path correctly in Bash on Windows. I don't use Windows, so I can't answer that from experience. From looking it up online it seems that `cd "/C/David Garcia"` might be right, but that's an educated guess. :slightly_smiling_face:
